Sources/VersionCheck/VersionCheck.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  VersionCheck.swift
+//  
+//
+//  Created by Emad Ghorbania on 11/11/2022.
+//
+
+import FirebaseRemoteConfig
+
+
+public struct VersionCheck {
+    fileprivate var remoteConfig: RemoteConfig
+    /// You should initiate this struct by injecting variables to be able to start using ``VersionCheck`` and it Validates the curent version of the app!
+    /// - Warning:You should configure FirebaseApp in your project by injecting info.plist to them before using this module
+    /// - Example:
+    /// ````
+    ///    do {
+    ///        let validation = try await VersionCheck.validateVersion(
+    ///        currentVersionKey: "iOSCurrentVersionTest",
+    ///        minSupportedVersionKey: "iOSMinSupportedVersionTest",
+    ///        currentAppVersion: Bundle.main.versionString
+    ///    )
+    ///    switch validation {
+    ///        case .mustUpdate :
+    ///            mustUpdateAlert = true
+    ///        case .updateExist :
+    ///            updateAlert = true
+    ///        case .noUpdate:
+    ///         No Update
+    ///        }
+    ///    } catch {
+    ///        Send error to crashlytics
+    ///    }
+    ///````
+    /// - Parameters:
+    ///   - currentVersionKey: The key has been stored in Firebase Remote Config
```

Comment:
What if we changed this line to:"The Firebase Remote Config key for current app version"?
Same for `minSupportedVersionKey`?

Reply:
done